common/src/main/java/com/microsoft/identity/common/internal/result/ILocalAuthenticationResult.java

```diff
@@ -114,4 +114,11 @@
      */
     @Nullable
     String getRefreshTokenAge();
+
+    @Nullable
```

Comment:
My OCD is going off... can we put the annotation after the javadoc to match the others?